cmd/chainsimulator/flags.go

```diff
@@ -96,6 +96,15 @@ var (
 		Usage: "This flag is used to specify the initial epoch when chain simulator will start",
 		Value: 0,
 	}
+	autoGenerateBlocks = cli.BoolFlag{
+		Name:  "auto-generate-blocks",
+		Usage: "Boolean option to specify that blocks should be generated automatically, after a given period of time",
```

Comment:
is the second part of the comment valid? "after a given period of time"? or it is referring that there is time between blocks?